exonum/src/sandbox/consensus/basic.rs

```diff
@@ -76,6 +76,19 @@ fn test_reach_one_height() {
     sandbox.assert_state(Height(2), Round(1));
 }
 
+/// idea of the test is to reach one height in the network with single validator
```

Comment:
Capital letter in the beginning and dot at the end :)

Reply:
Oh